saleor/tests/e2e/orders/taxes/test_order_calculate_simple_taxes_based_on_shipping_tax_class.py

```diff
@@ -74,39 +32,40 @@ def test_order_calculate_simple_tax_based_on_shipping_tax_class_CORE_2010(
         permission_manage_shipping,
         permission_manage_product_types_and_attributes,
         permission_manage_taxes,
+        permission_manage_settings,
         permission_manage_orders,
     ]
     assign_permissions(e2e_staff_api_client, permissions)
 
-    (
-        warehouse_id,
-        channel_id,
-        channel_slug,
-        shipping_method_id,
-    ) = prepare_shop(e2e_staff_api_client)
-
-    country_tax_rate, shipping_tax_rate, tax_class_id = prepare_tax_configuration(
+    shop_data = prepare_shop(
         e2e_staff_api_client,
-        channel_slug,
-        country_code="US",
         country_tax_rate=10,
-        shipping_tax_rate=8,
+        shipping_country_tax_rate=8,
         prices_entered_with_tax=True,
     )
-
+    channel_id = shop_data["channel_id"]
+    warehouse_id = shop_data["warehouse_id"]
+    shipping_price = shop_data["shipping_price"]
+    shipping_method_id = shop_data["shipping_method_id"]
+    shipping_country_tax_rate = shop_data["shipping_country_tax_rate"]
+    country_tax_rate = shop_data["country_tax_rate"]
+    country_tax_class_id = shop_data["country_tax_class_id"]
+    country = shop_data["country"]
+    update_country_tax_rates(
+        e2e_staff_api_client,
+        country,
+        [{"rate": shipping_country_tax_rate}],
+    )
     variant_price = "33.33"
-    (
-        _product_id,
-        product_variant_id,
-        product_variant_price,
-    ) = prepare_product(
+    (product_id, product_variant_id, product_variant_price) = prepare_product(
         e2e_staff_api_client,
         warehouse_id,
         channel_id,
         variant_price,
     )
-    shipping_tax_class = {"taxClass": tax_class_id}
-    update_shipping_price(e2e_staff_api_client, shipping_method_id, shipping_tax_class)
+
+    tax_class = {"taxClass": country_tax_class_id}
+    update_product(e2e_staff_api_client, product_id, tax_class)
```

Comment:
No need to update the product tax class, In this case tax rate for the product should be taken from the country rate 
product tax class = null
product type tax class = null
country rate = 10
shipping tax class = 8

product tax= variant price * country rate
shipping tax = shipping price * shipping tax class rate